Sources/Parsers/MarkdownParser.php

```diff
@@ -893,10 +893,8 @@ public function parse(string $string, bool $from_bbcode_parser = false, array $o
 	 */
 	public static function load(int $output_type = self::OUTPUT_HTML, ?int $hard_breaks = null): self
 	{
-		$hard_breaks = $hard_breaks ?? (int) (Config::$modSettings['markdown_brs'] ?? 0);
-
 		if (!isset(self::$parsers[$output_type][$hard_breaks])) {
-			self::$parsers[$output_type][$hard_breaks] = new self($output_type, $hard_breaks);
+			self::$parsers[$output_type][$hard_breaks] = new self($output_type);
 		}
 
 		return self::$parsers[$output_type][$hard_breaks];
```

Comment:
Please remove all references to `$hard_breaks` entirely.
1. Please remove the `$hard_breaks` input parameter and the documentation for it.
2. Change `self::$parsers[$output_type][$hard_breaks]` to just `self::$parsers[$output_type]` in all cases.